package.json

```diff
@@ -64,15 +64,18 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.2.0",
-    "@edx/frontend-build": "13.0.1",
     "@edx/reactifex": "^2.1.1",
+    "@openedx/frontend-build": "13.0.27",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "12.1.5",
     "axios-mock-adapter": "1.21.2",
     "glob": "7.2.3",
     "husky": "7.0.4",
     "jest": "27.5.1",
+    "postcss-loader": "^8.0.0",
     "prettier": "2.8.1",
-    "rosie": "2.1.0"
+    "rosie": "2.1.0",
+    "sass-loader": "^14.0.0",
+    "source-map-loader": "^5.0.0"
```

Comment:
I've narrowed it down to these 3

`postcss-loader`, `sass-loader`, and `source-map-loader`

`npm start` breaks without `postcss-loader` or `sass-loader`, but works without `source-map-loader`
`npm run build` breaks without `source-map-loader`

It's still not clear to me where the problem is living in the webpack config chain

Reply:
resolved by doing a fresh regen of `package-lock.json` (thanks @arbrandes!)